web-common/src/Halogen/Monaco.purs

```diff
@@ -151,7 +154,9 @@ handleAction settings Init = do
       let
         languageId = view Monaco._id settings.languageExtensionPoint
       liftEffect do
-        when (languageId == "typescript") $ Monaco.addExtraTypesScriptLibsJS monaco
+        when (languageId == "typescript") do
+          Monaco.addExtraTypesScriptLibsJS monaco
```

Comment:
spelling mistake here in the original function `addExtraTypesScriptLibsJS` should be `addExtraTypeScriptLibsJS`